app/src/main/java/io/gnosis/safe/di/TypeAdapters.kt

```diff
@@ -0,0 +1,66 @@
+package io.gnosis.safe.di
+
+import com.squareup.moshi.FromJson
+import com.squareup.moshi.JsonQualifier
+import com.squareup.moshi.ToJson
+import io.gnosis.safe.utils.parseToBigInteger
+import pm.gnosis.crypto.utils.asEthereumAddressChecksumString
+import pm.gnosis.model.Solidity
+import pm.gnosis.models.Wei
+import pm.gnosis.utils.asEthereumAddress
+import pm.gnosis.utils.hexAsBigInteger
+import pm.gnosis.utils.toHexString
+import java.math.BigInteger
+
+class WeiAdapter {
```

Comment:
Should we use the ones in svalinn that @biafra23 added

Reply:
We would need a svalinn release first. I would postpone this. I can go over the app and replace its usage as soon as there is a release.

Reply:
But the PR for this change was already merged, wasn't it? If you want we can do a release for svalinn tomorrow and then I can replace this class with your changes.

Reply:
I created a release. The version is v0.12.0